cmd/builder/fake_command_test.go

```diff
@@ -35,3 +38,41 @@ func TestFakeCommand(t *testing.T) {
 		t.Errorf("failed to use mocked Exec function on FakeCommand")
 	}
 }
+
+func TestFakeFailedCommand(t *testing.T) {
+	f := &FakeFailedCommand{MockError: errors.New("error")}
+
+	f.AppendArgs("arg1", "arg2")
+
+	if !f.CalledAppendArgs || f.ArgsAppend == nil || f.ArgsAppend[0] != "arg1" || f.ArgsAppend[1] != "arg2" {
```

Comment:
we do not override the AppendArgs, so I don't think we need to test it again on FakeFailedCommand